python/paddle/nn/clip.py

```diff
@@ -12,9 +12,1088 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-# TODO: define the functions to clip gradient of parameter
-from ..fluid.clip import ClipGradByGlobalNorm  # noqa: F401
-from ..fluid.clip import ClipGradByNorm  # noqa: F401
-from ..fluid.clip import ClipGradByValue  # noqa: F401
+import copy
+import warnings
 
-__all__ = []
+import paddle
+import paddle.autograd as imperative_base
+from paddle import _C_ops, _legacy_C_ops
+from paddle.common_ops_import import Variable, check_type, default_main_program
+from paddle.fluid import core, framework, layers, unique_name
+from paddle.fluid.data_feeder import check_variable_and_dtype
+from paddle.framework import (
+    LayerHelper,
+    _in_legacy_dygraph,
+    _non_static_mode,
+    in_dygraph_mode,
+)
+from paddle.tensor.layer_function_generator import templatedoc
+
+__all__ = [
+    'set_gradient_clip',
+    'ErrorClipByValue',
+    'ClipGradByValue',
+    'ClipGradByNorm',
+    'ClipGradByGlobalNorm',
+]
```

Comment:
`ClipGradByValue`, `ClipGradByNorm` and `ClipGradByGlobalNorm` are already in \_\_all\_\_ of `paddle/nn/__init__.py`, so not need to be added here. `set_gradient_clip` and `ErrorClipByValue` should not be public API, only for internal use, also not need to be added here.

Reply:
done from your suggestion.  you can checkout detail in https://github.com/PaddlePaddle/Paddle/pull/48946/commits/fde5139d7371dc1b550591b857464e3dad1a2fa6